internal/expr/bindinputs.go

```diff
@@ -110,6 +141,13 @@ type typedInputExpr struct {
 	input typeinfo.Input
```

Comment:
Maybe worth updating the comment on this to somehow imply that doesn't include the input expressions in INSERT statements.

Reply:
I changed it to:
```
// typedInputExpr stores information about a Go value to use as a standalone query
// input.
```